app/views/layouts/application.html.erb

```diff
@@ -112,9 +112,13 @@
           <% end %>
       <% end %>
       <% if @current_user && @current_user.pseudonym.unique_id.downcase != 'teastudent'  %>
-          <div class="sm-menu-item dropdown">
+          <div class="sm-user-name">
+             <%= @current_user.name %>
+          </div>
+          <div class="dropdown">
             <a id="sm-profile-dropdown" href="/profile" type="button" aria-haspopup="true" data-toggle="dropdown" data-id="profile-dropdown" aria-owns="sm-profile-dropdown" aria-labelledby="type-filter-description" aria-expanded="false">
               <div class="sm-menu-item-icon-container" aria-hidden="true">
+
```

Comment:
Curious about this additional line insertion here, but not enough to block a pr.